R/tt_pos_and_access.R

```diff
@@ -939,42 +1003,49 @@ setMethod("[", c("VTableTree", "numeric", "numeric"),
           function(x, i, j, ..., drop = FALSE) {
     ## have to do it this way because we can't add an argument since we don't
     ## own the generic declaration
-    keep_topleft <- list(...)[["keep_topleft"]] ## returns NULL if not presesnt
+    keep_topleft <- list(...)[["keep_topleft"]]
```

Comment:
no reason not to do the `%||%` thing we do for all the other ones here... (and above)

Reply:
The problem here is that if we want to add the option to keep that information whenever `keep_topleft` is not set and all rows are selected, we need to keep the knowledge of not having it set until the first check (i.e. if all rows are selected). I am not against keeping it as a fixed default. In that case, though, I think it should be `TRUE`.